modules/mirror-registry-introduction.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-installation-images.adoc
+
+[id="mirror-registry-introduction_{context}"]
+== Mirror registry for {product-title} introduction
+
+For disconnected deployments of {product-title}, a container registry is required to carry out the installation of the clusters. If users want to run a production-grade registry service on such a cluster, a separate registry deployment is required to install the first cluster. The mirror registry addresses this need and is included in every OpenShift subscription. It is available for download on the {product-title} link:https://console.redhat.com/openshift/downloads[*Downloads*] page.
```

Comment:
We should talk with RCM, and possibly alter the 'link' here; as I think we can provide a customer portal link as well.